package.json

```diff
@@ -54,10 +54,9 @@
     "@babel/code-frame": "7.24.2",
     "@gemini-testing/commander": "2.15.3",
     "@jspm/core": "2.0.1",
-    "@types/mocha": "10.0.1",
     "@wdio/globals": "8.21.0",
     "@wdio/protocols": "8.22.0",
-    "@wdio/types": "8.21.0",
+    "@wdio/types": "8.39.0",
```

Comment:
I see mismatching versions of wdio packages (under the `@wdio` scope). I am 99% sure they need to be the same version (8.39.1 if you chose it for the main webdriverio package), even if everything works now, because it would lead to duplicating packages when user installs testplane